engine/schema/src/main/java/com/cloud/storage/dao/GuestOSHypervisorDaoImpl.java

```diff
@@ -157,7 +157,7 @@ public List<GuestOSHypervisorVO> listByOsNameAndHypervisorMinimumVersion(String
         sc.and(sc.entity().getGuestOsName(), SearchCriteria.Op.EQ, guestOsName);
         sc.and(sc.entity().getHypervisorType(), SearchCriteria.Op.EQ, hypervisorType);
         sc.and().op(sc.entity().getHypervisorVersion(), SearchCriteria.Op.GTEQ, minHypervisorVersion);
-        sc.or(sc.entity().getHypervisorVersion(), SearchCriteria.Op.NEQ, "default");
+        sc.or(sc.entity().getHypervisorVersion(), SearchCriteria.Op.EQ, "default");
```

Comment:
should this be GTEQ?

Reply:
no, default is not part of the simver and it is OR'ed so I think EQ is good